### ISSUE TITLE
Can you explain more on how to run this?

### ISSUE THREAD

**Issue by joon2730** [junhyeok-dev/JLVEA#1]
I saw and tried the command ./JLVEA-cpp [encrypt|decrypt] FILENAME SEED you suggested, but it didn't work and probably I'm doing it wrong.

I hope you to explain about it more on readme and think few examples would do help me a lot.

**Comment by junhyeok-dev** [comment]
Hi,

Thank you for trying to use JLVEA encryption mechanism.

Before you use the JLVEA encryptor, you should compile the source and generate executable binary.

Please follow the installation instruction on README first.

If you using other operating systems such as Windows or macOS, install gcc, cmake and execute

```
cmake CMakeLists.txt
make
```

As compile has completed, now you can encrypt or decrypt video file with JLVEA encryptor/decryptor.

You can encrypt the video 'example.mp4' with a seed number 12345678 with the command,

```
./JLVEA-cpp encrypt example.mp4 12345678
```

If you want to decrypt the encrypted video, just change the parameter 'encrypt' to 'decrypt'.

Feel free to comment if you need more details or having problem.

**Comment by joon2730** [comment]
Hi, thank you for answering my question.

I managed to run the program and encrypted a mp4 video file, but now I can't find where the encrypted file has been created and the original file has not been changed as well.

I also discovered some errors in your source code while debugging it.
It should include fstream at the header, and the conditionals for determining mode(encrypt and decrypt) are flawed.
